frontend/src/toolbar/elements/heatmapLogic.ts

```diff
@@ -515,6 +461,41 @@ export const heatmapLogic = kea<heatmapLogicType>([
     })),
 
     listeners(({ actions, values }) => ({
+        fetchHeatmapApi: async () => {
+            const { href, wildcardHref } = values
+            const { date_from, date_to } = values.commonFilters
+            const { type, aggregation } = values.heatmapFilters
+            const urlExact = wildcardHref === href ? href : undefined
+            const urlRegex = wildcardHref !== href ? wildcardHref : undefined
+
+            // toolbar fetch collapses queryparams but this URL has multiple with the same name
```

Comment:
```suggestion

```

Reply:
IIRC I've fixed that so we don't need to care

Reply:
nope... https://github.com/PostHog/posthog/blob/0d11b570216d42b616bd6bf2a78c38dcdbf6c556/posthog/api/element.py#L135

wild, I _remember_ doing it - I'm losing my mind